tests/NRF24L01_Create.cc

```diff
@@ -79,7 +79,7 @@ TEST(NRF24L01_Create, pass_null_pointer_ddr_ce)
             &port_ce,
             pin_ce);
 
-    EXPECT_TRUE(NULL == handle);
+    EXPECT_TRUE(NULL == handle);3
```

Comment:
[Test] Please remove this magic number '3' and please add comment for this test.